src/pijama_ast/src/lib.rs

```diff
@@ -129,31 +165,45 @@ impl<'a> Display for Primitive {
     }
 }
 
-/// Encapsulates a conditional where "if `cond` then `body`"
+/// Encapsulates a conditional statement in Pijama's syntax of the 
+/// form "if `cond` then `body`".
 #[derive(Debug, Eq, PartialEq)]
 pub struct Branch<'a> {
+    /// The conditional part of the Branch that is checked for truthiness.
     pub cond: Located<Block<'a>>,
+    /// The body of the Branch that is executed if `cond` is truthy.
```

Comment:
I'd change "truthy" by "true". This could lead someone to think we support stuff like: `if 1 do 1 else 0 end`.